src/bank.rs

```diff
@@ -254,14 +273,13 @@ impl LiteSVM {
         account.set_executable(true);
         account.set_data_from_slice(program_bytes);
 
-        let loaded_program = solana_bpf_loader_program::load_program_from_bytes(
-            false,
+        let mut loaded_program = solana_bpf_loader_program::load_program_from_bytes(
             Some(self.log_collector.clone()),
             &mut LoadProgramMetrics::default(),
             account.data(),
             account.owner(),
             account.data().len(),
-            self.slot,
+            0,
```

Comment:
Why do you set the loaded program start slot to 0?

Reply:
Oh yeah I ran into this issue where the program isn't automatically available in the same slot as it was deployed. But I am changing it now to use self.slot for effective_slot and deployment_slot, which fixes the issue